makefile.mingw

```diff
@@ -190,12 +190,12 @@ src/prngs/rc4.o src/prngs/rng_get_bytes.o src/prngs/rng_make_prng.o src/prngs/so
 src/prngs/sprng.o src/prngs/yarrow.o src/stream/chacha/chacha_crypt.o src/stream/chacha/chacha_done.o \
 src/stream/chacha/chacha_ivctr32.o src/stream/chacha/chacha_ivctr64.o \
 src/stream/chacha/chacha_keystream.o src/stream/chacha/chacha_setup.o src/stream/chacha/chacha_test.o \
-src/stream/rc4/rc4_stream.o src/stream/rc4/rc4_test.o src/stream/salsa20/salsa20_crypt.o \
-src/stream/salsa20/salsa20_done.o src/stream/salsa20/salsa20_ivctr64.o \
-src/stream/salsa20/salsa20_keystream.o src/stream/salsa20/salsa20_setup.o \
-src/stream/salsa20/salsa20_test.o src/stream/sober128/sober128_stream.o \
-src/stream/sober128/sober128_test.o src/stream/sosemanuk/sosemanuk.o \
-src/stream/sosemanuk/sosemanuk_test.o
+src/stream/rabbit/rabbit.o src/stream/rc4/rc4_stream.o src/stream/rc4/rc4_test.o \
```

Comment:
wouldnt it make more sense to add the only new file in an own line, instead of juggling all the items in 6 different lines around ? that makes it hard to see what was actually changed

Reply:
changes to OBJECTS in the makefiles are automated and also checked in travis, therefore you can ignore them

Reply:
if that is so, maybe it'd be cleaner to have those autogenerated changes in a single file which is included from all the other makefiles

Reply:
I also already thought of that, but 

1. this can't be used from the `makefile.msvc` as that one needs `.obj` file endings
2. this can't be used from `makefile.unix` AFAIK posix make doesn't support the `include` statement

which leaves us with `makefile.mingw` and `makefile_include.mk` which could use it... yeah it'd be one file less... then we should probably think if it'd make sense to migrate the `makefile.mingw` to be able to use `makefile_include.mk`... (which should be doable)

Reply:
maybe you could use a variable for the extension, e.g. rabbit.$(OEXT)

Reply:
we can simply keep just one file name per line like:
```
...
src/stream/salsa20/salsa20_crypt.o \
src/stream/salsa20/salsa20_done.o \
src/stream/salsa20/salsa20_ivctr64.o \
src/stream/salsa20/salsa20_keystream.o \
src/stream/salsa20/salsa20_setup.o \
src/stream/salsa20/salsa20_test.o \
...
```

then the diffs will be quite easy to read, but it will be quite a lot of lines

Reply:
> wouldnt it make more sense to add the only new file in an own line

Maybe it is not obvious but the object files list is updated via script with algorithm that sorts all obj files and prints them so that every line is not longer than 100 chars (it always was like this). If anybody has a better idea it won't be hard to implement.

Reply:
> but it will be quite a lot of lines

currently more than 420 lines... I'm against this, even 

> maybe you could use a variable for the extension, e.g. rabbit.$(OEXT)

true, then we could use this for all makefiles besides makefile.unix ...

Reply:
i'm assuming makefile.unix is for BSD make compatibility. don't they support variable expansion too ?

Reply:
[posix make](http://pubs.opengroup.org/onlinepubs/009695399/utilities/make.html) supports variable expansion but no include statement

Reply:
also I don't think it's a priority to change some automated task, so someone can better review it...

Reply:
makefile.unix also works with IBM's make (on AIX) and HP's make (on HP-UX) both of them are neither GNU make nor BSD make

IMO we should move make(file) related ideas into a separate issue/pr